extra/exports

```diff
@@ -0,0 +1,464 @@
+# Symbols exported by the main Tarantool executable.
+#
+# Keep the symbols sorted by name for search and addition
+# simplicity, to avoid duplicates. Makes no much sense to
+# split them into any sections. Anyway some symbols will
+# end-up belonging to several of them at once, and it
+# would require moving lines here when a symbol goes from
+# privately exported to the public C API.
```

Comment:
I sorted them programmatically and there were a lot of corrections. Please, try to sort them non-manually. Either write a small script or for instance use an online tool: https://www.ipvoid.com/sort-text-lines/.

Reply:
I used `sort` for sorting. I deleted the comments and then I do
```
┌─[leonid@vasya-hp]─[~/work/mail/tarantool/extra]
└──╼ cat exports | sort | uniq > exports_sort
┌─[leonid@vasya-hp]─[~/work/mail/tarantool/extra]
└──╼ diff exports exports_sort 
┌─[leonid@vasya-hp]─[~/work/mail/tarantool/extra]
└──╼ 
```
Why is `sort` a bad way or where am I going wrong?

Reply:
Don't know how it works, but I see the issues even without any tools. For example, according to your current sorting, you have `lua_isuserdata` < `luaL_checkstack` < `lua_next`. How is it possible? Obviously, all `lua_*` functions should be together - they have exactly the same prefix. But in your patch they are split in half by some `luaL_*` functions.

Reply:
Yep. AFAIU for `sort` the `_` is nothing. But I think this is a good way because it is simple (just do `sort` after you added several symbols) and we use the same we to sort curl symbols.
I'll try googling how to make the underline meaningful.

Reply:
I tried to  use `LC_COLLATE="C" sort -f` to sort. IMHO the result is ok.

Reply:
Nice, now looks good and works on my machine too.